README.md

```diff
@@ -29,9 +29,8 @@ Within this **open-ce** repository, you will find:
 conda_build_config.yaml:  This is a YAML config file which contains all package
     prerequisite information accumulated from across the Open-CE components
 open-ce/
-    This directory contains the parent build scripts, most notably
-    `build_env.py` and `build_feedstock.py`.
-    `build_env.py` is the script you will use to build Open-CE packages.
+    This directory contains the build and validation tool `open-ce`
```

Comment:
Add a period at the end of the line.

Reply:
I added your suggested change above to do this.